www/modloader/_oneloader_configuration.js

```diff
@@ -7,6 +7,20 @@ if (window.nw.App.argv[0] !== "test") {
         "ogg": { "encrypt": "rpgmaker", "target_extension": "rpgmvo" }
     };
 
+    let LANGUAGE = "en";
+    try {
+        const fs = require("fs");
+        eval(fs.readFileSync("www/js/plugins.js", "utf8"))
```

Comment:
Avoid evaluating files like this raw without any wrapper. You do clean up by setting $plugins to undefined at the end, but it's better to never write to that in the first place.
( https://github.com/rphsoftware/OneLoader/blob/f8735624e88e9361c93a64cc2bbcfd82b9f028ca/www/mods/oneloader/oneloader-assistant.js#L390 for an example).